apps/dashboard/src/components/primitives/control-input/variable-plugin/variable-pill-widget.ts

```diff
@@ -144,7 +157,15 @@ export class VariablePillWidget extends WidgetType {
 
       span.addEventListener('mouseenter', () => {
         if (!this.tooltipElement) {
-          this.tooltipElement = this.renderTooltip(span);
+          const issues = this.getVariableIssues();
+          const firstIssue = issues[0];
+
+          this.tooltipElement = this.renderTooltip({
```

Comment:
refactored the way we show the tooltip, now it will be positioned top center and will appear/hide with the tw animation like the original tooltip has